showcase/build/iaf_cond_alpha_implicit.h

```diff
@@ -0,0 +1,888 @@
+/* generated by template org.nest.nestml.neuron.NeuronHeader*/
+/*
+*  iaf_cond_alpha_implicit.h
+*
+*  This file is part of NEST.
+*
+*  Copyright (C) 2004 The NEST Initiative
+*
+*  NEST is free software: you can redistribute it and/or modify
+*  it under the terms of the GNU General Public License as published by
+*  the Free Software Foundation, either version 2 of the License, or
+*  (at your option) any later version.
+*
+*  NEST is distributed in the hope that it will be useful,
+*  but WITHOUT ANY WARRANTY; without even the implied warranty of
+*  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+*  GNU General Public License for more details.
+*
+*  You should have received a copy of the GNU General Public License
+*  along with NEST.  If not, see <http://www.gnu.org/licenses/>.
+*
+*/
+#ifndef IAF_COND_ALPHA_IMPLICIT
+#define IAF_COND_ALPHA_IMPLICIT
+#include "config.h"
+
+#ifdef HAVE_GSL
+#include <gsl/gsl_errno.h>
+#include <gsl/gsl_matrix.h>
+#include <gsl/gsl_odeiv.h>
+// forwards the declaration of the function
```

Comment:
I thinks this should read ``// forwards declaration of the function`` (no "the"), though this is not of the utmost importance.